fmriprep/data/utils.py

```diff
@@ -23,6 +23,14 @@
 import subprocess as sp
 from fmriprep.data.compat import _urllib, md5_hash
 
+try:
```

Comment:
I generally prefer this solution http://python-future.org/compatible_idioms.html#urllib-module:
```
from future.standard_library import install_aliases
install_aliases()

from urllib.parse import urlparse, urlencode
from urllib.request import urlopen, Request
from urllib.error import HTTPError
```